doc/Installation/Install-LibreNMS.md

```diff
@@ -0,0 +1,560 @@
+
+**Please note the minimum supported PHP version is 7.2.5**
+
+# Install Required Packages
+
+=== "Ubuntu 20.04"
+    === "NGINX"
+        ```
+        apt install software-properties-common
+        add-apt-repository universe
+        apt update
+        apt install acl curl composer fping git graphviz imagemagick mariadb-client mariadb-server mtr-tiny nginx-full nmap php7.4-cli php7.4-curl php7.4-fpm php7.4-gd php7.4-json php7.4-mbstring php7.4-mysql php7.4-snmp php7.4-xml php7.4-zip rrdtool snmp snmpd whois unzip python3-pymysql python3-dotenv python3-redis python3-setuptools
+        ```
+    
+    === "Apache"
+        ```
+        apt install software-properties-common
+        add-apt-repository universe
+        apt update
+        apt install acl curl apache2 composer fping git graphviz imagemagick libapache2-mod-fcgid mariadb-client mariadb-server mtr-tiny nmap php7.4-cli php7.4-curl php7.4-fpm php7.4-gd php7.4-json php7.4-mbstring php7.4-mysql php7.4-snmp php7.4-xml php7.4-zip rrdtool snmp snmpd whois python3-pymysql python3-dotenv python3-redis python3-setuptools
+        ```
+
+=== "CentOS 8"
+    === "NGINX"
+        ```
+        yum install epel-release
+        yum install cronie fping git ImageMagick mariadb mariadb-server mtr net-snmp net-snmp-utils nginx nmap php-fpm php-cli php-common php-curl php-gd php-json php-mbstring php-process php-snmp php-xml php-zip php-mysqlnd python3-PyMySQL python3-redis python3-memcached rrdtool
+        ```
+    
+    === "Apache"
+        ```
+        yum install epel-release
+        yum install git cronie fping jwhois ImageMagick mtr MySQL-python net-snmp net-snmp-utils nmap python-memcached rrdtool policycoreutils-python httpd mariadb mariadb-server unzip python3 python3-pip
+        ```
+
+=== "Debian 10"
+    === "NGINX"
+        ```
+        apt install curl composer fping git graphviz imagemagick mariadb-client mariadb-server mtr-tiny nginx-full nmap php7.3-cli php7.3-curl php7.3-fpm php7.3-gd php7.3-json php7.3-mbstring php7.3-mysql php7.3-snmp php7.3-xml php7.3-zip python-memcache python-mysqldb rrdtool snmp snmpd whois python3-pymysql python3-dotenv python3-redis python3-setuptools
+        ```
+        
+# Add librenms user
+
+```
+useradd librenms -d /opt/librenms -M -r -s /usr/bin/bash
+```
+
+# Download LibreNMS
+
+```
+cd /opt
+git clone https://github.com/librenms/librenms.git
+```
+
+# Set permissions
+
+```
+chown -R librenms:librenms /opt/librenms
+chmod 771 /opt/librenms
+setfacl -d -m g::rwx /opt/librenms/rrd /opt/librenms/logs /opt/librenms/bootstrap/cache/ /opt/librenms/storage/
+setfacl -R -m g::rwx /opt/librenms/rrd /opt/librenms/logs /opt/librenms/bootstrap/cache/ /opt/librenms/storage/
+```
+
+# Install PHP dependencies
+
+```
+su - librenms
+./scripts/composer_wrapper.php install --no-dev
+exit
+```
+
+# DB Server
+
+## Configure MySQL
+
+=== "Ubuntu 20.04 / Debian 10"
+    ```
+    vi /etc/mysql/mariadb.conf.d/50-server.cnf
```

Comment:
I would say it's better we stop writing what editor to use, or use $EDITOR

Reply:
I plan to check that "edit" is available on all platforms.  That references the $EDITOR env var.

Reply:
CentOS does not, open to suggestions.  I support vi or nano (which is more friendly to use)

Reply:
I suggest just writing "edit the file", ie. As a text and not a command. If you don't know how to edit a file, you're going to have a bad time anyway :(

Reply:
hmm, that removes the simplicity of just copy and pasting the box (with the nice button)

Reply:
why not simply "open `/etc/mysql/mariadb.conf.d/50-server.cnf`" and "add/update whatever"

Reply:
$ open /etc/mysql/mariadb.conf.d/50-server.cnf
bash: open: command not found

Reply:
I agree with @murrant The nice thing about lnms installs have always been the copy and paste functionality. Especially for newcomers. I would keep the `vi` but that is just my 2c.